lms/djangoapps/ccx/views.py

```diff
@@ -348,28 +382,49 @@ def visit(node, depth=1):
         widgets, which use text inputs.
         Visits students visible nodes only; nodes children of hidden ones
         are skipped as well.
+
+        - Section has only start date. Due date does not applicable for it. If ccx section do
```

Comment:
@pdpinch Can you or someone from the MITx team help Amir craft docstrings that are a bit more readable? I'm confused by these, sorry.